src/components/App/App.js

```diff
@@ -12,25 +12,34 @@ class App extends Component {
     super()
     this.state = {
       cities: [],
-      error: '',
+      searchedCities: [],
+      renderingSearch: false,
```

Comment:
I just realized that I never deleted this line of code!  No more tracking the state of rendering.

Reply:
We should really have searchedCities: [] in the SearchBar component too! Maybe we can refactor if we have time.